app/hid_dap_bridge.c

```diff
@@ -0,0 +1,74 @@
+#include <FreeRTOS.h>
+#include <DAP.h>
+#include <stdbool.h>
+#include <task.h>
+#include <queue.h>
+
+#include "platform/platform.h"
+#include "drivers/max14662.h"
+#include "platform/usb/hid.h"
+#include "hid_dap_bridge.h"
+
+#define TASK_STACK_SIZE		512
+#define TASK_NAME			"HIDdap"
+#define TASK_PRIORITY		1
+
+static struct {
+	bool initialized;
+	StackType_t task_stack[TASK_STACK_SIZE];
+	TaskHandle_t task_handle;
+	StaticTask_t task_tcb;
+} self;
+
+static void task_worker(void *p_arg)
+{
+	uint8_t resp_buf[USB_FS_MAX_PACKET_SIZE];
```

Comment:
wouldn't it be saner to have this buffer (And the one below probably) as file-level variables so they aren't allocated on the stack? Then the stack doesn't need to be 2KB either probably.